SourceCode/chaos_hw_02/chaos_hw_02.cpp

```diff
@@ -0,0 +1,84 @@
+﻿#include <iostream>
+#include <fstream>
+#include <vector>
+#include <cstdlib>
+#include <ctime>
+
+
+void CreateGridImage(const int numRectY, const int numRectX, const int maxColorComponent, const int rectHeight, const int rectWidth, std::vector<std::vector<std::vector<int>>>& image)
+{
+    for (int y = 0; y < numRectY; ++y) {
```

Comment:
Instead of using 4 nested for loops, you could directly check at each pixel in which smaller triangle it falls